weblogic-azure-aks/src/main/arm/scripts/setupDBConnections.sh

```diff
@@ -3,112 +3,75 @@
 
 echo "Script ${0} starts"
 
-# read <dbPassword> <wlsPassword> from stdin
-function read_sensitive_parameters_from_stdin() {
-    read dbPassword wlsPassword
-}
-
-#Function to display usage message
-function usage() {
-    usage=$(cat <<-END
-Usage:
-echo <dbPassword> <wlsPassword> |
-    ./setupDBConnections.sh
-    <aksClusterRGName>
-    <aksClusterName>
-    <databaseType>
-    <dbUser>
-    <dsConnectionURL>
-    <jdbcDataSourceName>
-    <wlsDomainUID>
-    <wlsUser>
-    <dbOptType>
-END
-)
-    echo_stdout "${usage}"
```

Comment:
While I support the idea of no longer reading a very long list of parameters from stdin, and instead using environment variables, I do not support losing the helpful usage message.  Please restore the usage message and make it clear what environment variables this script requires to function correctly.